Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -152,6 +155,23 @@ public static class GoogleAuthConsts
         /// </summary>
         internal static string EffectiveMetadataServerUrl => GetEffectiveMetadataUrl(null, DefaultMetadataServerUrl);
 
+        /// <summary>
+        /// Retry strategy recommended for the OAuth2 token endpoints.
+        /// </summary>
+        internal static readonly ExponentialBackOffInitializer OAuth2TokenEndpointRecommendedRetry = new ExponentialBackOffInitializer(
+            ExponentialBackOffPolicy.RecommendedOrDefault,
+            () => new BackOffHandler(new BackOffHandler.Initializer(new ExponentialBackOff(deltaBackOffPercent: 10, maximumNumOfRetries: 3))
+            {
+                // Give the back-off strategy control over how much to wait for.
+                MaxTimeSpan = TimeSpan.MaxValue,
+                HandleExceptionFunc = ex => false,
+                HandleUnsuccessfulResponseFunc = response =>
+                    response.StatusCode == HttpStatusCode.RequestTimeout || // 408
+                    (int) response.StatusCode == 429 || // 429 Enum value not availbale in .NET Standard 2.0 or .NET Framework 4.6.2
```

Comment:
availbale => available

Reply:
And add a comment saying what the enum value would be - I don't know what 429 means offhand.

Reply:
Both done